ivadomed/main.py

```diff
@@ -44,18 +45,22 @@ def cmd_train(context):
         torch.cuda.set_device(gpu_number)
         print("Using GPU number {}".format(gpu_number))
 
-    # Boolean which determines if the selected architecture is FiLMedUnet or Unet or MixupUnet
+    # Boolean which determines if the selected architecture is FiLMed-Unet or Unet or Mixup-Unet
     metadata_bool = False if context["metadata"] == "without" else True
     film_bool = (bool(sum(context["film_layers"])) and metadata_bool)
 
     unet_3D = context["unet_3D"]
+    HeMIS = context['HeMIS']
     attention = context["attention_unet"]
-    HeMIS = context['missing_modality']
     if film_bool:
         context["multichannel"] = False
         HeMIS = False
     elif context["multichannel"]:
         HeMIS = False
+    if HeMIS:
+        # Initializing the missing probability for HeMIS.
+        # The lower the probability the higher, the more modalities are missing.
```

Comment:
"The lower the probability, the more modalities are missing." ?

Reply:
I agree the comment is not clear, but I believe it means Higher probability, means more modalities missing. I'll reprhase it ;)